bugsnag-plugin-react-native/src/main/java/com/bugsnag/android/DeviceDeserializer.java

```diff
@@ -7,11 +7,14 @@
 
     @Override
     public DeviceWithState deserialize(Map<String, Object> map) {
+        Number num = MapUtils.getOrNull(map, "apiLevel");
+        Integer apiLevel = num != null ? num.intValue() : null;
```

Comment:
Worth pushing this (and getLong) into MapUtils?

Reply:
Extracted to MapUtils class.